src/app/core/admin/admin-entity-details/admin-entity-field/configure-entity-field-validator/configure-entity-field-validator.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, EventEmitter, Input, Output } from "@angular/core";
+
+import { NgForOf } from "@angular/common";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatInputModule } from "@angular/material/input";
+import {
+  FormBuilder,
+  FormGroup,
+  FormsModule,
+  ReactiveFormsModule,
+} from "@angular/forms";
+import { DialogCloseComponent } from "../../../../common-components/dialog-close/dialog-close.component";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { MatButtonModule } from "@angular/material/button";
+import { EntityConstructor } from "../../../../entity/model/entity";
+import { MatCheckboxModule } from "@angular/material/checkbox";
+import { EntitySchemaField } from "app/core/entity/schema/entity-schema-field";
+import { DynamicValidator } from "app/core/common-components/entity-form/dynamic-form-validators/form-validator-config";
+
+@Component({
+  selector: "app-configure-entity-field-validator",
+  standalone: true,
+  imports: [
+    NgForOf,
+    MatFormFieldModule,
+    MatInputModule,
+    DialogCloseComponent,
+    FormsModule,
+    MatCheckboxModule,
+    FontAwesomeModule,
+    ReactiveFormsModule,
+    MatButtonModule,
+  ],
+  templateUrl: "./configure-entity-field-validator.component.html",
+  styleUrl: "./configure-entity-field-validator.component.scss",
+})
+export class ConfigureValidatorPopupComponent {
+  fieldId: string;
```

Comment:
not used? Can you go through the component and check whether all the inputs and properties are actually required for you final implementation?